packages/bitcore-wallet-client/lib/common/constants.js

```diff
@@ -34,6 +39,28 @@ Constants.UNITS = {
       minDecimals: 2,
     }
   },
+  bch: {
```

Comment:
Copay now passes in coin instead of wallet.settings.unitCode. Previously BTC was hard coded and set as default for bch formatAmount(). Added BCH and ETH as vaild coin params for formatAmount()